lib/goliath/rack/formatters/json.rb

```diff
@@ -19,7 +19,7 @@ def post_process(env, status, headers, body)
         end
 
         def json_response?(headers)
-          headers['Content-Type'] =~ %r{^application/(json|javascript)}
+          headers['Content-Type'] =~ %r{^application/(json|javascript|vnd\.api\+json)}
```

Comment:
Happy to make that change -- this version is a little verbose, but felt more readable.